mitiq/zne/scaling/folding.py

```diff
@@ -540,176 +536,6 @@ def _apply_fold_mask(
     return folded_circuit
 
 
-@accept_qprogram_and_validate
-def fold_gates_from_left(
```

Comment:
We could keep the signature of these two methods and replace the body with just raising a `DeprecationWarning`. Just in case a user is using those directly.